Port mod_celadon

Ykipss opened the pull request:
Постарался как мог перенести все моды старого целадона с минимумом изменений корней билда
В код срали-мазали, мы тоже насрали и намазали. Я в основном был вооружён #define'ом в .dme файле, поэтому его нужно добавить что-б избежать 300+ ошибок

Ykipss committed:
mod_celadon

Постарался как мог перенести все моды старого целадона с минимумом изменений корей билда

Ykipss committed:
fix

Исправлено всё то, о чём мне сказал Войко

Ykipss committed:
Update shiptest.dme

Исправил инклюды 

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update shiptest.dme

доделал дмешник, закомментировал вот енто вот всё что мне не надо было

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
dmi fix

дал имя состоянию в файлике

Ykipss committed:
Update ipc_accessories.dmi

Я рот ебал этих файлов

Ykipss committed:
Смерть неверным дмишкам

Я выпилил нахуй .dmi файлы вызывающие ошибки при компиле в гите

Ykipss committed:
Merge branch 'beta-dev' into mod_celadon

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
comments

Комментарии добавил к корням, отсылающим где надо на модуль

Ykipss committed:
Merge branch 'mod_celadon' of https://github.com/Ykipss/Shiptest into mod_celadon

MrCat15352 commented on mod_celadon/code/game/mecha/combat/gygax.dm:
```diff
@@ -0,0 +1,81 @@
+	wreckage = /obj/structure/mecha_wreckage/gygax/proto
+
+/obj/mecha/combat/gygax/alt
+	desc = "Алтерьнативная модель лёгкого экзоскелета.."
```
Лишняя точка в описании

MrCat15352 commented on mod_celadon/code/game/mecha/combat/gygax.dm:
```diff
@@ -0,0 +1,81 @@
+	wreckage = /obj/structure/mecha_wreckage/gygax/white
+
+/obj/mecha/combat/gygax/proto
+	desc = "Прототип лёгкого боевого экзоскелета.."
```
Лишняя точка в описании

MrCat15352 commented on mod_celadon/code/game/mecha/equipment/tools/work_tools.dm:
```diff
@@ -0,0 +1,182 @@
+	icon_state = "ripleypubupg"
+	source_mech = list(/obj/mecha/working/ripley/mkii)
+	result_mech = /obj/mecha/working/ripley/mkii/ripley_zairjah
+
```
Лишняя строка

MrCat15352 commented on shiptest.dme:
```diff
@@ -880,7 +880,7 @@
 #include "code\game\machinery\medipen_refiller.dm"
 #include "code\game\machinery\navbeacon.dm"
-#include "code\game\machinery\newscaster.dm"
+//#include "code\game\machinery\newscaster.dm"
```
Почему отключено? Причина?

MrCat15352 commented on shiptest.dme:
```diff
@@ -3373,7 +3373,7 @@
 #include "code\modules\surgery\bodyparts\robot_bodyparts.dm"
 #include "code\modules\surgery\bodyparts\species_parts\ethereal_bodyparts.dm"
-#include "code\modules\surgery\bodyparts\species_parts\ipc_bodyparts.dm"
+//#include "code\modules\surgery\bodyparts\species_parts\ipc_bodyparts.dm"
```
Почему отключено? Причина?

MrCat15352 commented on shiptest.dme:
```diff
@@ -2753,7 +2753,7 @@
 #include "code\modules\mob\living\simple_animal\hostile\mining_mobs\elites\herald.dm"
 #include "code\modules\mob\living\simple_animal\hostile\mining_mobs\elites\legionnaire.dm"
-#include "code\modules\mob\living\simple_animal\hostile\mining_mobs\elites\pandora.dm"
+//#include "code\modules\mob\living\simple_animal\hostile\mining_mobs\elites\pandora.dm"
```
Отключено, потому что пандора сломаная

MrCat15352 commented on mod_celadon/updates/SlightPvEOverhaul/elite_pandora/wizardconstructs.dm:
```diff
@@ -0,0 +1,182 @@
+/////////////////Juggernaut///////////////
```
Добавить комментарий типа "Сломано, починить"

MrCat15352 commented on mod_celadon/updates/SlightPvEOverhaul/elite_pandora/pandora.dm:
```diff
@@ -0,0 +1,370 @@
+#define SINGULAR_SHOT 1
```
Добавить комментарий "Сломано, нужен фикс"

MrCat15352 commented on mod_celadon/mod_celadon.dme:
```diff
@@ -1 +1,38 @@
+#include "code\modules\research\designs\mechfabricator_designs.dm"
+#include "code\modules\surgery\bodyparts\species_parts\ipc_bodyparts.dm"
+#include "code\modules\vending\pizzamat.dm"
 #include "__tweaks/fullscreen.dm"
```
Поменять слеш

Ykipss committed:
Update elite.dm

хуй

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

MrCat15352 commented on mod_celadon/code/game/objects/items/storage/belt.dm:
```diff
@@ -0,0 +1,36 @@
+	supports_variations = VOX_VARIATION
+
+/obj/item/storage/belt/security/webbing/elisium/vest_black
+	name = "Полная разгрузка , черная."
```
Убрать пробел

MrCat15352 reviewed (changes requested):
Поправить

Ykipss committed:
Update gygax.dm

убрал точки

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update work_tools.dm

Убрал строку лишнюю(лол)

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update belt.dm

Пробел убрал лишний(лол)

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update pandora.dm

Добавил в комментарий-описание строку с сообщением что поломано и нужен фикс

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update wizardconstructs.dm

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

Ykipss committed:
Update pandora.dm

Signed-off-by: Snr Bydlo <109545437+Ykipss@users.noreply.github.com>

MrCat15352 reviewed (commented):
Проверил. Вроде норм. Не должно сильно поломаться

MrCat15352 reviewed (approved):
Одобрено

Ykipss committed:
последний, надеюсь, коммит

/commit suicide(ОСУЖДАЮ). Короче с пандорой я проебался там всё нормально, я её вернул взад.

MrCat15352 reviewed (approved):
Теперь все точно хорошо.

MrCat15352 commented:
Все хорошо
